### ISSUE TITLE
inform user about major changes in the mobile apps

### ISSUE THREAD

**Issue by richlv** [mapillary/mapillary_issues#2358]
the ios mobile app was recently upgraded to have auto/manual modes. this is real great, but maybe it is worth showing a popup when the user launches a new version for the very first time - explaining about the major changes in functionality.
for example, it looks like the auto mode now does not allow setting exposure point while it is running, but allows taking a manual photo instead. this is not a very intuitive change for existing users without some small help :)

**Comment by millenbop** [comment]
I agree completely. We need to introduce some kind of "what's new" system in the app(s).

(I think you figured it out, but to set the exposure point you can only do it before starting the capture)

**Comment by richlv** [comment]
yeah, after a few images with wrong exposure ;)
i know that other apps do show a popup (actually it just is a full-screen info page) about new stuff - i think maps.me does that as an example (where they have a shiny page and you can slide between news items). it would be enough just borrowing that approach :)

**Comment by millenbop** [comment]
Yup, something like that would work.

**Comment by Kaldea** [comment]
_mapillary_issues_ is no longer an active repository.

Let’s continue discussing features in our forum! We will import this issue as topic to the [Mapillary Forum](http://forum.mapillary.com) in the coming weeks. We’ll notify here once it’s been done. See you there!
